### ISSUE TITLE
PDF viewer doesn't work with zotero extension

### ISSUE THREAD

**Issue by LiYunyang** [gdh1995/vimium-c#1212]
**Describe the bug**

Zotero doesn't response when PDF viewer for Vimium C is working. This was previously fixed by the author (https://forums.zotero.org/discussion/83913/save-to-zotero-on-a-web-page-of-the-pdf-viewer-extension), but now the extension is not working again.

**Browser, OS and Vimium C versions**

- Browser name:  Chrome
- Browser version: 128.0.6613.120
- Vimium C version: latest PDF viewer extension
- OS name and version: macOS 14.6.1

**Comment by gdh1995** [comment]
Do you mean it is still broken even when you keep `prefer keeping URL in address bar unchanged` enabled?

I'm not sure what happened. And, sorry but I have no much time to test the extension in recent years.

**Comment by LiYunyang** [comment]
That's right. The url in the browser looks like `chrome-extension://nacjakoppgmdcpemlfnfegmlhipddanj/https://arxiv.org/pdf/xxx` (instead of the expected 'https://arxiv.org/pdf/xxx'?)
It may be unrelated, but Chrome warns, "This extension may soon no longer be supported." Will this extension continue? It has been really helpful for the past years, and I am very sorry it will be discontinued.
